src/nupic/bindings/reals.i

```diff
@@ -66,7 +66,6 @@ def GetNumpyDataType(typeName):
   if typeName == "NTA_Real": return numpy.float64
   elif typeName == "NTA_Real32": return numpy.float32
   elif typeName == "NTA_Real64": return numpy.float64
-  #elif typeName == "NTA_Real128": return numpy.float128
   else: raise RuntimeError("Unsupported type name.")
```

Comment:
Not required for this PR, but would help with diagnostics to include typeName in the RuntimeError message.